doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -634,6 +634,88 @@ must be sent.
 *default location*, then the query will be declared as ambigous and an HTTP error response with code ``409``
 must be sent. 
 
+## Notification messages
+
+Notifications include two fields:
+
+* `subscriptionId` which represents the concerned subscription that originates the notification
+* `data` an array with the notification data itself which includes the entity and attributes concerned. Each
+  element in the array corresponds to a different entity. The way of representing each entity depends on
+  the `attrsAs` parameter associated to the subscription, as shown below.
+
+
+If `attrsAs` is `normalized` then default entity representation is used:
+
```

Comment:
If we use the term 'normalized' for the default entity representation, then the correct property name should be:

'attrsFormat' (as you suggested in previous PR), with three possible values ['normalized', 'keyValues', 'values']. 

In the absence of the 'attrsFormat' property, the normalized mode will be used. 

if attrsFormat takes any other value an error will be raised. 

I think we need to add those clarifications.